src/views/ArtistList.jsx

```diff
@@ -44,6 +49,9 @@ const ArtistList = () => {
               <Link to={`/artists/${id}`}>{name}</Link>
             </div>
           ))}
+          {currentUser ? (
+            <CreateArtist addArtist={addArtist} />
+          ) : <p>Log in to create a new artist</p>}
```

Comment:
Para mantener el estilo de este operador ternario:
```suggestion
          ) : (
            <p>Log in to create a new artist</p>
          )}
```